[REFACTOR] improve service history formatting

lucasatdriano opened the pull request.

lucasatdriano committed:
[REFACTOR] improve service history formatting

LucasBonato commented on src/main/java/com/serv/oeste/domain/entities/service/Service.java:
```diff
@@ -87,13 +87,25 @@ public Service(Integer id, String equipamento, String marca, String filial, Stri
+            formattedSituation = "ABERTURA: " + formattedSituation;
+        }
+
+        String newEntry = String.format(
```
Isso daqui pela regra de negócio não pode ser trocada a ordem, a data tem que vir primeiro, depois a situação formatada e ai sim a descriação

lucasatdriano committed:
[FIX] equipment history filter

- Update ServiceSpecifications to accept ClientEntity/TechnicianEntity
- Add missing brand filter

lucasatdriano committed:
Merge branch 'main' of https://github.com/LucasBonato/ServOeste into patch-3
